src/kind_lab_to_nwb/rat_behavioural_phenotyping_2025/water_maze/add_trials.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
+from pynwb import NWBFile
+
+
+def add_trials_to_nwbfile(
+    nwbfile: NWBFile,
+    trials: pd.DataFrame,
+):
```

Comment:
I'm changing this to a behavior data interface to make it consistent with the other tasks